_pages/en/strategy-build-vs-buy.md

```diff
@@ -37,66 +44,140 @@ th {
+- [Introduction](#introduction)
+  - [Purpose](#purpose)
+  - [Target Audience](#target-audience)
+  - [Business Case](#business-case)
+- [Guiding Policy](#guiding-policy)
+  - [Governance, Compliance, and Reporting](#governance-compliance-and-reporting)
+  - [Finance](#finance)
+  - [Architecture](#architecture)
+- [Coherent set of actions](#coherent-set-of-actions)
+- [Measuring the Strategy's success](#measuring-the-strategys-success)
+- [Appendix A - Business Case (Diagnostic)](#appendix-a---business-case-diagnostic)
+- [Appendix B - Traceability Matrix](#appendix-b---traceability-matrix)
+- [Appendix D - Definitions](#appendix-d---definitions)
+- [Appendix C - Context](#appendix-c---context)
+- [Appendix E - References](#appendix-e---references)
+
+## Introduction
+
+### Purpose
+
+To provide IITB with a build versus buy assessment framework.
+
+The strategy includes:
+
+1. A **guiding policy**, which serves to set automatic decisions that defines the build versus buy assessment framework
+2. A **coherent set of actions** (an action plan), which serve to operationalize the build versus buy assessment framework.
+
+The intent behind this strategy is to communicate a decision by the CIO (not yet approved) on a path forward (the Guiding Policy), and what investments are needed to operationalize that decision (the coherent set of actions).
+
+### Target Audience
+
+This **strategy document** is targeted to stakeholders involved in determining whether to build or to buy software components.
+More specifically, stakeholders involved in architecting solutions, defining technology standards, developing, delivering, operationalizing and maintaining IT Solutions (such as Enterprise Architecture, Enterprise Operations, etc.).
+This includes both IITB and non-IITB stakeholders (such as from Business Functions, SSPB, CDO, ISB, IAERMB, and SSC).
+The list of stakeholders are listed in section [Coherent set of actions](#coherent-set-of-actions) and are expected to participate in the execution stage of this strategy necessary to operationalize the Guiding Policy.
+
+The **Guiding Policy**, once operationalized, will target stakeholders involved when the need for an IT Solution is raised, when the IT Solution is conceptualized, architected, developed, delivered, operationalized, and maintained.
+All ESDC personnel involved in the exploration of IT Solutions and in IT investment decisions are expected to adhere to this policy.
+
+### Business Case
+
+Moving to the digital age requires improving IT's responsiveness and promoting a shared accountability with ESDC stakeholders on the use of technology.
+
+To improve IT's responsiveness, we must find ways to reduce risks associated with its use as well as with the ways we obtain it.
```

Comment:
To improve IT's responsiveness, we must find ways to reduce risks associated with its use as well as with the ways we obtain it.
=>
To improve IT's responsiveness, ways must be found to reduce risks associated with its use the approaches with which it is acquired.

Reply:
fixed in commit 43496b1dd48905c31deb23a41ac7ca03fa42323a